packages/fxa-content-server/app/tests/spec/lib/router.js

```diff
@@ -513,32 +513,33 @@ describe('lib/router', () => {
   });
 
   describe('React-related methods', () => {
-    beforeEach(() => {
-      const mockAlwaysShownReactGroups = [
-        'simpleRoutes',
-        'resetPasswordRoutes',
-      ];
-      sinon
-        .stub(router, 'ALWAYS_SHOWN_REACT_GROUPS')
-        .value(mockAlwaysShownReactGroups);
-    });
     describe('showReactApp', () => {
       beforeEach(() => {
         sinon.spy(router, 'showReactApp');
       });
 
       describe('returns true', () => {
-        it('when routeName is included in react group route and user is in experiment', () => {
+        // must be tested with a route that is not fully rolled out to production but that should serve React when in experiment
```

Comment:
Is it possible to modify the mock config a bit closer so that we're consistently testing the cases laid out? I'm wondering if we can do that by altering `config = { showReactApp: showReactAppAll };` - the tests in this file have been a bit fragile since they were made.

Reply:
Modified to stub in mocks for the reactRouteGroups, so that we're now testing expected behaviour without relying on route definitions that might change as we roll out individual route groups to production.